### ISSUE TITLE
Offsets: Play with data

### ISSUE THREAD

**Issue by LindsayMacfarlane** [bcgov/cas-review#38]
This card is about:

- accessing the data
- storing it
- evaluate quality of data
- make it available in Metabase?

#### Checklist:
- [x] get API documentation from BCCR
- [x] download all the data or a subset of it
- [ ] put it in a metabase instance (not completed)
- [ ] start looking at common keys with swrs data (not completed)
- [x] document insights/questions gathered during this phase of exploration (e.g., data warehousing miro board)

#### Notes:
- [ ] Documentation stored in Scrum Team>CPR>BC Carbon Registry and Offsets > BCCR API Documentation
- [ ] Exploring the data comes in a second stage (We may need a ticket for this)

**Comment by pbastia** [comment]
Things we can start with:
- downloading all the data or a subset of it
- put it in a metabase instance
- start looking at common keys with swrs data

**Comment by pbastia** [comment]
This is pending on the team receiving access to the data, expected for Jan 31

**Comment by pbastia** [comment]
@dleard we can also time-box this one

**Comment by dleard** [comment]
^ good idea

**Comment by Dianadec** [comment]
I know this is already in the sprint backlog, but seeing as how it is tagged with Backlog Refinement and the card description is limited, what other information should we include here? Can take a look at this during Tuesday Refinement this week. @dleard @pbastia cc @LindsayMacfarlane

**Comment by LindsayMacfarlane** [comment]
Hi @dleard - will this ticket be wrapped up by end of sprint? Do you foresee any new tasks that we should create new tickets for for next sprint?

**Comment by dleard** [comment]
yes, I think it can be done by the end of the week. I've downloaded the data and recorded the shape in miro, I've also been writing a rudimentary ETL to grab the data and put it into tables in a postgres database. There isn't a lot to glean from the public API data as far as how it could relate to our current datasets. We mentioned in the IHS meeting that we'd at least like to get a data schema that includes identifiers not available from the public API, a follow up on that would be good as those identifiers are probably what's going to be really helpful to us.

**Comment by LindsayMacfarlane** [comment]
Thanks @dleard! I've sent a follow up email to BCCR requesting that information.

**Comment by dleard** [comment]
@LindsayMacfarlane I'm adding this ticket to the Pending PO column and we can go over what I've learned before moving it to closed.

**Comment by LindsayMacfarlane** [comment]
The last 3 checklists couldn't be completed based on the information in the dataset. We should wait for the next set of data/info before we try setting up our own database/metabase instance (Dylan indicates that it would be a lot of effort for low value at this stage). I'll pull the bullets into a new ticket for future consideration.
